docs/sdks/android/methods.md

```diff
@@ -148,9 +148,17 @@ CleverPush.getInstance(this).getAvailableTags(tags -> {
 
 Set<String> subscribedTagIds = CleverPush.getInstance(this).getSubscriptionTags();
 
-CleverPush.getInstance(this).addSubscriptionTag("tag_id");
+// add single tag
+CleverPush.getInstance(this).addSubscriptionTag("tag_id")
+
+// add multiple tags
+CleverPush.getInstance(this).addMultipleSubscriptionTags(new  String[] {});
```

Comment:
- only 1 space after `new` 
- add 2 example tags in array (TAG_ID_1, TAG_ID_2)

Reply:
Also please (in the SDK and here) rename the Method to "addSubscriptionTags". Remove the word "Multiple" so we have same naming on iOS and Android